Readme.md

```diff
@@ -109,51 +186,69 @@ if (true)
 }
 ```
 
-Also, notice the use of whitespace before and after the condition statement.
-
-## Method chaining
-
-One method per line should be used if you want to chain methods.
+#### Newline character after open braces
 
-You should also indent these methods so it's easier to tell they are part of the same chain.
+Always insert a line break following the open brace `{` character, except for
+object literals or immediately closed brace `}` character.
 
 *Right:*
 
 ```js
-User
-  .findOne({ name: 'foo' })
-  .populate('bar')
-  .exec(function(err, user) {
-    return true;
-  });
-````
+function emptyFunc() {}
+
+function trivialFunc() {
+  return false;
+}
+
+if (trivialBoolean) {
+  return false;
+} else {
+  doSomething();
+}
+
+var obj = {'a': 1, 'b': 2};
+```
 
 *Wrong:*
 
 ```js
-User
-.findOne({ name: 'foo' })
-.populate('bar')
-.exec(function(err, user) {
-  return true;
-});
+function trivialFunc() { return false; }
 
-User.findOne({ name: 'foo' })
-  .populate('bar')
-  .exec(function(err, user) {
-    return true;
-  });
+if (trivialBoolean) {  return false; }
```

Comment:
I think this case is an exception. We always need to check for errors and commonly do this `if (err) { return cb(err); }`. Since it's so common, I think cases like this are ok to keep on one line. 

For longer errors, split to multiple lines.